grammar.js

```diff
@@ -61,6 +61,7 @@ module.exports = grammar({
     [$.sized_type_specifier],
     [$.attributed_statement],
     [$._declaration_modifiers, $.attributed_statement],
+    [$.enum_specifier],
```

Comment:
might be nice each time we have a conflict to explain where it comes from, with an example of ambiguity (and how GLR will automatically solve the ambiguity).